Update Stormworks

delphyinium opened the pull request:
# Description
This is a fix that was tested in the issue mentioned #1851 

Changes:

- change image from wine_latest to wine_staging
- update readme to address console output and tps misreporting
- updated install script to add echo message and removed unneeded package install step
closes https://github.com/parkervcp/eggs/issues/1851

<!-- Please explain what is being changed or added as a short overview for this PR. Also, link existing relevant issues if they exist with resolves # -->

## Checklist for all submissions

<!-- insert X into the brackets to mark it as done. You can click preview to make the links appear clickable. -->

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

QuintenQVD0 reviewed (changes requested):
Use the panel export feature. You dont edit the json file directly. Then the egg will update to pltv_2 format if your panel is up to date

delphyinium commented:
> Use the panel export feature. You dont edit the json file directly. Then the egg will update to pltv_2 format if your panel is up to date

Apologies. Fixed. I went through the effort of deleting the file and uploading the new one when I could have just edited the existing file 🤦 
Should be good now though.

QuintenQVD0 commented:
This egg needs a extra variable to auto update steamcmd. And in the install script can be upate as the apt update and the apt install is not needed because it uses the install image that has all those packages

delphyinium commented:
> This egg needs a extra variable to auto update steamcmd. And in the install script can be upate as the apt update and the apt install is not needed because it uses the install image that has all those packages. And the new echo done at the end of install script. If you don't want to do that I will do that for you tommorow.

I believe I have fixed everything. I tested Auto-Update to verify it checks for an update, which it does. I also got rid of the apt install, since it is not required like you said, and added an echo at the end to signal it has completed installation.

QuintenQVD0 commented on game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json:
```diff
@@ -49,6 +49,16 @@
+            "default_value": "0",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:20",
```
This should be `required|boolean`

QuintenQVD0 commented on game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json:
```diff
@@ -0,0 +1,54 @@
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:wine_staging": "ghcr.io\/parkervcp\/yolks:wine_staging"
```
Please use `Wine_staging|<url>` in the panel docker image so then it when it export it only shows wine staging to the end user.

QuintenQVD0 commented:
If the eggs then can be tested by someone or I will do it in the weekend then I will aprove this.

delphyinium commented:
> If the eggs then can be tested by someone or I will do it in the weekend then I will aprove this. 

Ok, I can fix these in a bit, then I'll test and get back to you.

delphyinium commented:
Sorry for the delay, I quickly made the changes you requested. I tested and it seems to be running smooth. Once again apologies for the delay.

QuintenQVD0 commented:
Np, LGTM!
